xdem/coreg.py

```diff
@@ -1568,627 +1354,592 @@ def _to_matrix_func(self) -> NDArrayf:
 
             return transform_mgr.get_transform(0, len(self.pipeline))
 
-    def __iter__(self) -> Generator[Coreg, None, None]:
-        """Iterate over the pipeline steps."""
-        yield from self.pipeline
 
-    def __add__(self, other: list[Coreg] | Coreg | CoregPipeline) -> CoregPipeline:
-        """Append Coreg(s) or a CoregPipeline to the pipeline."""
-        if not isinstance(other, Coreg):
-            other = list(other)
-        else:
-            other = [other]
-
-        pipelines = self.pipeline + other
-
-        return CoregPipeline(pipelines)
+class BlockwiseCoreg(Coreg):
```

Comment:
Same as `Pipeline`, this should probably be renamed to something without "Coreg" in it.

Reply:
Same as above, will bring it up in summary reply below!